plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveSplit.java

```diff
@@ -250,6 +261,7 @@ public Object getInfo()
                 .put("partitionName", partitionName)
                 .put("deserializerClassName", getDeserializerClassName(schema))
                 .put("s3SelectPushdownEnabled", s3SelectPushdownEnabled)
+                .put("startingRowId", initialRowId)
```

Comment:
keep string and variable name consistent

Reply:
ohh I didn't notice this one, this was initial name of the field than I did renaming, good catch, thanks